<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureSettings</class>
 <widget class="QWidget" name="MeasureSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
  background-color: rgb(39, 39, 48);
  color: rgb(190, 190, 190);
  font: 75 10pt &quot;DejaVu Sans&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>40</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hLayout_titleBar">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel {
  font-weight: bold;
  font-size: 13px;
  color: #ffffff;
}
</string>
         </property>
         <property name="text">
          <string>Measure</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblChanName">
         <property name="styleSheet">
          <string notr="true">QLabel {
  font-weight: normal;
  font-size: 14px;
  color: #ffffff;
}
</string>
         </property>
         <property name="text">
          <string>Channel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 114, 0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gLayout_Measurements">
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>16</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel {
  font-weight: normal;
  font-size: 14px;
  color: rgba(255, 255, 255, 153);
}
</string>
         </property>
         <property name="text">
          <string>Counter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel {
  font-weight: normal;
  font-size: 14px;
  color: rgba(255, 255, 255, 153);
}
</string>
         </property>
         <property name="text">
          <string>Display All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_Counter">
         <property name="styleSheet">
          <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 100px;
  max-width: 100px;
}

QPushButton:checked {
  border-image: url(:/icons/switch_on.png);
}
QPushButton:!checked {
  border-image: url(:/icons/switch_off.png);
}

QPushButton:checked:disabled {
  border-image: url(:/icons/switch_on_disabled.png);
}
QPushButton:!checked:disabled {
  border-image: url(:/icons/switch_off_disabled.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_measDisplayAll">
         <property name="styleSheet">
          <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 100px;
  max-width: 100px;
}

QPushButton:checked {
  border-image: url(:/icons/switch_on.png);
}
QPushButton:!checked {
  border-image: url(:/icons/switch_off.png);
}

QPushButton:checked:disabled {
  border-image: url(:/icons/switch_on_disabled.png);
}
QPushButton:!checked:disabled {
  border-image: url(:/icons/switch_off_disabled.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="button_measDeleteAll">
         <property name="styleSheet">
          <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 240px;
  max-width: 240px;

  border: 0px;
  border-radius: 4px;

  background-color: #4a64ff;
  color: #ffffff;

  font-size: 14px;
}

QPushButton:pressed {
  background-color: #2a44df;
}

QPushButton:disabled {
  background-color:  #727273;
}</string>
         </property>
         <property name="text">
          <string>Delete All</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gLayout_Statistics">
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>16</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">QLabel {
  font-weight: normal;
  font-size: 14px;
  color: rgba(255, 255, 255, 153);
}
</string>
         </property>
         <property name="text">
          <string>Statistics</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_StatisticsEn">
         <property name="styleSheet">
          <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 100px;
  max-width: 100px;
}

QPushButton:checked {
  border-image: url(:/icons/switch_on.png);
}
QPushButton:!checked {
  border-image: url(:/icons/switch_off.png);
}

QPushButton:checked:disabled {
  border-image: url(:/icons/switch_on_disabled.png);
}
QPushButton:!checked:disabled {
  border-image: url(:/icons/switch_off_disabled.png);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">QLabel {
  font-weight: normal;
  font-size: 14px;
  color: rgba(255, 255, 255, 153);
}
</string>
         </property>
         <property name="text">
          <string>Statistics</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_StatisticsReset">
         <property name="styleSheet">
          <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 100px;
  max-width: 100px;

  border: 0px;
  border-radius: 4px;

  background-color: #4a64ff;
  color: #ffffff;

  font-size: 14px;
}

QPushButton:pressed {
  background-color: #2a44df;
}</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout_hMeasurements">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>44</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayout_vMeasurements">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
